workbench/tests/binary_tests/puppet.test.js

```diff
@@ -232,11 +235,13 @@ test('Check local userguide links', async () => {
   const downloadModal = await page.waitForSelector('.modal-dialog');
   const downloadModalCancel = await downloadModal.waitForSelector(
     'aria/[name="Cancel"][role="button"]');
+  await page.waitForTimeout(500); // waiting for click handler to be ready
```

Comment:
Any reason this one is longer than the others? I don't think it's necessary if different click events could need different wait times but would it be convenient to have a constant variable defining these wait values?

Reply:
I don't know why it's different so I've made it the same, and I added a variable.

But it's likely each button is different, in terms of when it becomes functional. It seems dependent on whether the javascript containing the click handler's function has loaded yet, or whether the element is visible yet, or otherwise visually blocked somehow. All we know for sure is that the element is present in the DOM (that's what the `waitForSelector` accomplishes). But these other things remain unknown.